### PR TITLE
Jonas feature3

### PR DESCRIPTION
**Pull request by JonasPetraska**

### CODE REVIEW

File: src/components/CalendarView.jsx [hunk 1]
```diff
@@ -33,108 +35,139 @@ class CalendarView extends React.Component {
...
-						</p>
-						
-						
-						<div className="flex-spacer" />
-						<button className="primary margin-top-24">Keisti</button>
-						
+								</li>
+							</ul>
+
+							<h3 className="margin-top-16">Mokymosi dienos komentaras:</h3>
```
Comment (confiktura) [inline src/components/CalendarView.jsx:155]: This should be translated

File: src/components/CalendarView.jsx [hunk 2]
```diff
@@ -33,108 +35,139 @@ class CalendarView extends React.Component {
...
+						</div>
+
+						<div className="flex-spacer" />
+
+						<div className="cal-side-panel flex-down">
+							<h1 className="center unbold">
+								{formatDate(this.state.day)}
+							</h1>
+
+							<h3>Mokymosi dienos temos:</h3>
```
Comment (vodpox) [inline src/components/CalendarView.jsx:127]: This should be translated

**Comment by JonasPetraska** [comment]
Thanks. Added those translations.

File: src/components/Role.jsx [hunk 3]
```diff
@@ -60,25 +60,33 @@ class Role extends React.Component {
...
 	renderRoleUsers() {
-		if (this.state.usersList && this.state.usersList.length > 0) {
-			return (
-				<div>
-					<h3 className="margin-top-24">Visų vartotojų sąrašas:</h3>
+		if (this.state.usersList) {
+			if (this.state.usersList.length > 0) {
+				return (
+					<div>
+						<h3 className="margin-top-24">Visų vartotojų sąrašas:</h3>
```
Comment (vodpox) [inline src/components/Role.jsx:67]: one more untranslated string

**Pull request merged**